Libplanet/Net/NetMQTransport.cs

```diff
@@ -246,12 +241,10 @@ public async Task StartAsync(CancellationToken cancellationToken)
                 );
                 _endPoint = new DnsEndPoint(turnEp.Address.ToString(), turnEp.Port);
 
-                // FIXME should be parameterized
-                tasks.Add(BindingProxies(_cancellationToken));
-                tasks.Add(BindingProxies(_cancellationToken));
-                tasks.Add(BindingProxies(_cancellationToken));
+                var tasks = BindingMultipleProxies(_cancellationToken, _listenPort.Value, 3);
```

Comment:
Oops, I've comment wrong place maybe. 😅 

```suggestion
                List<Task> tasks = BindingMultipleProxies(_cancellationToken, _listenPort.Value, 3);
```